src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -297,7 +306,8 @@ private static CommandLine parseCli(String... args)
         }
         // ensure there is no conflicting options
         else if (cmdLine.hasOption(OPTION_T_NAME) || cmdLine.hasOption(OPTION_CAPITAL_T_NAME)
-                || cmdLine.hasOption(OPTION_J_NAME) || cmdLine.hasOption(OPTION_CAPITAL_J_NAME)) {
+                || cmdLine.hasOption(OPTION_J_NAME) || cmdLine.hasOption(OPTION_CAPITAL_J_NAME)
+                || cmdLine.hasOption(OPTION_S_NAME)) {
```

Comment:
> `else if (filesToProcess.size() > 1) {`
`result.add("Printing AST is allowed for only one file.");`

This error doesn't make sense to me with option S as we are not printing an AST tree.

Reply:
Done